cloudbuild.yaml

```diff
@@ -129,91 +121,6 @@ steps:
-    -var=jupyter_service_account=jupyter-sa-4-rag-$SHORT_SHA-$_BUILD_ID \
-    -var=cloudsql_instance=pgvector-instance-$SHORT_SHA-$_BUILD_ID \
-    -auto-approve -no-color
-    echo "pass" > /workspace/rag_tf_result.txt
-
-    # Validate Ray: Make sure pods are running
-    kubectl wait --for=condition=Ready pods -n rag-$SHORT_SHA-$_BUILD_ID -l 'component!=continuous-image-puller' --timeout=1200s
-    kubectl port-forward -n rag-$SHORT_SHA-$_BUILD_ID service/ray-cluster-kuberay-head-svc 8262:8265 &
-    # Wait port-forwarding to take its place
-    sleep 5s
-
-    # Validate Ray: Check dashboard
-    ray job submit --working-dir ./tests \
-    --address=http://127.0.0.1:8262 -- python -c "import ray; ray.init(); print(ray.cluster_resources())"
-    echo "pass" > /workspace/rag_ray_dashboard_result.txt
-
-    # Validate JupyterHub: Get hub url
-    kubectl get services -n rag-$SHORT_SHA-$_BUILD_ID
-    kubectl port-forward -n rag-$SHORT_SHA-$_BUILD_ID service/proxy-public 9443:80 &
-    # Wait port-forwarding to take its place
-    sleep 5s
-
-    # Validate JupyterHub: Test Hub
-    cd /workspace/modules/jupyter/tests
-    python3 test_hub.py "127.0.0.1:9443" $_AUTOPILOT_CLUSTER
-    echo "pass" > /workspace/rag_jupyterhub_test_result.txt
-
-    # Validate RAG: Test rag frontend
-    kubectl port-forward -n rag-$SHORT_SHA-$_BUILD_ID service/rag-frontend 8081:8080 &
-    # Wait port-forwarding to take its place
-    sleep 5s
-
-    cd /workspace/applications/rag/tests
-    python3 test_frontend.py "127.0.0.1:8081"
-    echo "pass" > /workspace/rag_frontend_result.txt
-
-    cd /workspace/
-    sed -i "s/<username>/$$KAGGLE_USERNAME/g" ./applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb
-    sed -i "s/<token>/$$KAGGLE_KEY/g" ./applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb
-    gsutil cp ./applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/
-    kubectl exec -it -n rag-$SHORT_SHA-$_BUILD_ID $(kubectl get pod -l app=jupyterhub,component=hub -n rag-$SHORT_SHA-$_BUILD_ID -o jsonpath="{.items[0].metadata.name}") -- jupyterhub token admin --log-level=CRITICAL | xargs python3 ./applications/rag/notebook_starter.py
-    # Wait for jupyterhub to trigger notebook pod startup
-    sleep 5s
-    kubectl wait --for=condition=Ready pod/jupyter-admin -n rag-$SHORT_SHA-$_BUILD_ID --timeout=500s
-    kubectl exec -it -n rag-$SHORT_SHA-$_BUILD_ID jupyter-admin -c notebook -- jupyter nbconvert --to script /data/rag-kaggle-ray-sql-interactive.ipynb
-    kubectl exec -it -n rag-$SHORT_SHA-$_BUILD_ID jupyter-admin -c notebook -- ipython /data/rag-kaggle-ray-sql-interactive.py
-
-    python3 ./applications/rag/tests/test_rag.py "http://127.0.0.1:8081/prompt"
-    echo "pass" > /workspace/rag_prompt_result.txt
-
-  allowFailure: true
-  waitFor: ['create gke cluster']
-
 - id: 'Generate Kubeconfig'
```

Comment:
It looks like this step and the following steps / tests are running on the rag cluster. Do we still need these tests?

Reply:
Adding the note that is looks like the same cluster is used for ray and rag, so these may still be needed until ray and the cluster creation are also removed.